### PR TITLE
Config flow documentation for html5

### PR DESCRIPTION
**Pull request by alexyao2015**
## Proposed change

Removes old configuration documentation for HTML5 because it is now using config flow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/112806
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a user-friendly configuration flow for HTML5 push notifications, simplifying the setup process.
  
- **Improvements**
	- Removed the manual configuration requirements and added a reference to a centralized configuration guide, enhancing user experience and reducing errors.

### CODE REVIEW

File: source/_integrations/html5.markdown [hunk 1]
```diff
@@ -58,15 +31,9 @@ The `html5` platform can only function if all of the following requirements are
 - You have configured SSL/TLS for your Home Assistant. It doesn't need to be configured in Home Assistant though, e.g., you can be running NGINX in front of Home Assistant and this will still work. The certificate must be trustworthy (i.e., not self-signed).
 - You are willing to accept the notification permission in your browser.
 
-### Configuring the platform
+### Add the HTML5 integration to Home Assistant
```
Comment (frenck) [inline source/_integrations/html5.markdown]: This header is already provided by the snippet below

```suggestion
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by joostlek — CHANGES_REQUESTED** [review]
There's a merge conflict

**Comment by alexyao2015** [comment]
Merge conflict is fixed.

**Review by joostlek — APPROVED** [review]

**Comment by joostlek** [comment]
Many thanks for your effort ❤️

**Pull request merged**